.github/workflows/e2e.yml

```diff
@@ -31,45 +31,70 @@ jobs:
       # test runtime environment, tests invoke rustc/cargo
       SYSTEM_TEST_RUST_TOOLCHAIN_VERSION: stable
 
-      # sets the version of soroban-js-client used by tests
-      SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION: "0.9.2"
+      # set the version of js-soroban-client to use, need to choose one of either 
+      # resolution options, using npm release or a gh ref:
+      #
+      # option #1, set the version of soroban-js-client based on a npm release version
+      SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION: 0.11.0
```

Comment:
this isn't compatible with the latest version of Core - it's intended to still support futurenet :/ you could try [soroban-client#121](https://github.com/stellar/js-soroban-client/pull/121) but I have no idea which xdr ref the core build uses so that might not work either. if you tell me the ref I can prep a branch heh

Reply:
ok, I thought that had the upgrade-base-xdr branch folded in which had the newer(er) xdr, ok, the version of core used here is `35170ec6c543ecc492e0ade8b32671cebd4df985` and it uses protocol-next ref on stellar-xdr of  [1894f09](https://github.com/stellar/stellar-core/tree/35170ec6c543ecc492e0ade8b32671cebd4df985/src/protocol-next), which looks same as the latest for [go repo ref for xdr-next](https://github.com/stellar/go/blob/master/Makefile#L18) , can we get a js-soroban-client ref based on that same xdr ver if it's not already?

Reply:
Ok! Working on it

Reply:
This is done here: https://github.com/stellar/js-soroban-client/commit/f06d7d35199f4967ff7a608c69efb9ad7d744be2!

Reply:
ok, I pulled in the reference here. once the tests get past contract install, then js-soroban-client would start getting used and can see how it goes with this now dated version of core.

..it looks like newer xdr is out and getting rebuilt on go repo per draft pr [go#5023](https://github.com/stellar/go/pull/5023) as of earlier today, which it is using new stellar-xdr ref of:
7015193879e9da7cb70d889a49eecf494fefe7e1

soroban-tools will take that go ref soon after, is it worthwhile to just use the js pr's now to update all the way to this same xdr also? and e2e will just need to wait for the dust to settle and new build/pull refs are available, this may have been what @2opremio was thinking as he had temporarily disabled e2e workflow earlier today.

Reply:
yeah I saw that it was getting bumped again. there is also https://github.com/stellar/stellar-xdr/pull/137 coming down the pipeline, so I'm totally fine with just waiting until they finally pinky swear that xdr is frozen before doing soroban-client / e2e updates, but also am not the one to make that call :man_shrugging: happy to also just keep regenerating whatever xdr ref is helpful for you haha